default/stringtables/en.txt

```diff
@@ -5047,6 +5056,69 @@ SITREP_EMPIRE_ELIMINATED_LABEL
+
+BEGINNER_HINT_08
+An Outpost ship can setup an Outpost on any unclaimed visible planet or asteroid that it can reach. Outposts can be settled by any species in your empire that has a connection to it, at least 5 happiness, and 3 population.
+
+BEGINNER_HINT_09
+Colony ships take longer to build than outpost ships but can settle planets outside of your supply range.
+
+BEGINNER_HINT_10
+Right click on a planet to access the Planet Suitability report. This will tell you what species from your empire will be able to successfully settle that planet. Numbers in green show the maximum population that the planet could achieve if colonized by that species, red numbers indicate that the species would die out if they attempted to colonize.
+
+BEGINNER_HINT_11
+Maximum populations that planets can achieve can be boosted by researching various growth techs, or by utilizing various rare specials that can be found on some planets.
+
+BEGINNER_HINT_12
+Be on the lookout for specials on planets like [[encyclopedia GROWTH_FOCUS_TITLE]]', they usually indicate some special resource on that planet that may benefit your entire empire if utilized properly. Right click on a special's icon to access its 'Pedia entry and find out more!
+
+BEGINNER_HINT_13
+Only [[species SP_EXOBOT]] can settle asteroids. If you find a lot of asteroids belts nearby you might want to research how to build them sooner than later!
+
+BEGINNER_HINT_14
+Research [[encyclopedia SUPPLY_TITLE]] improvements to increase the distance from your systems that your supply line reaches.
+
+BEGINNER_HINT_15
+Most species start out with a [[encyclopedia HAPPINESS_TITLE]] of 1 when you capture their planet, and this will normally increase by 1 per turn. Look out for [[encyclopedia XENOPHOBIC_SPECIES_TITLE]] species though! They don't like being near other species and this will effect the happiness and other values of both them and the other nearby species.
+
+BEGINNER_HINT_16
+You may want to hold off on researching [[tech LRN_PSIONICS]] until your empire includes a species with [[encyclopedia TELEPATHIC_TITLE]].
+
+BEGINNER_HINT_17
+[[encyclopedia SHIELDS_TITLE]] on ships defend against every shot that hits the ship, reducing the damage by the amount of the shield value. Ship shield values can be affected by various factors including space storms, what type of star system you are in, or if you are close to a captured empire capital planet.
+
+BEGINNER_HINT_18
+[[encyclopedia FOCUS_TITLE]] - pay attention to the focus setting of new planets that are added to your empire, either from settling them or from conquering them. This controls if the majority of the planet focuses on industrial production or on research or on defense. Later in the game more options become available as you research new technologies. Not all species are able to focus on [[encyclopedia RESEARCH_TITLE]] or [[encyclopedia INDUSTRY_TITLE]].
```

Comment:
"the majority" is misleading / vague

comma after later in the game

more focus options may unlock as new technologies are researched, and different species may have different focus options available.